newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -492,6 +584,15 @@ const PrivateAssetPackInformationPage = ({
       ]
     : [];
 
+  const canRedeemAssetPack = canRedeemProduct({
+    redeemConditions: privateAssetPackListingData.redeemConditions,
+    subscription,
+  });
+
+  const showPersonalUsageRedemptionBanner = shouldDisplayCalloutToGetSubscriptionOrClaimAssetPack(
```

Comment:
name to update I guess (not personal anymore)

Reply:
would it make more sense for the `usageType` of the pack that can be claimed to come from the backend?
Even if it would always be the same for now, `commercial`, at least we can control it in the future (+ the label below can adapt depending on which one it is)